static/app/views/performance/transactionSummary/teamKeyTransactionButton.tsx

```diff
@@ -0,0 +1,53 @@
+import styled from '@emotion/styled';
+
+import Button from 'app/components/button';
+import TeamKeyTransaction, {
+  TitleProps,
+} from 'app/components/performance/teamKeyTransaction';
+import {IconStar} from 'app/icons';
+import {t, tn} from 'app/locale';
+import {Organization, Team} from 'app/types';
+import EventView from 'app/utils/discover/eventView';
+import withTeams from 'app/utils/withTeams';
+
+type Props = {
+  eventView: EventView;
+  organization: Organization;
+  teams: Team[];
+  transactionName: string;
+};
+
+function TeamKeyTransactionButton({eventView, teams, ...props}: Props) {
+  if (eventView.project.length !== 1) {
```

Comment:
`project = [-1]`? I also wish that property wasn't singular lol.

As an aside we should probably make a 'isSingleProject' util function or function on the EventView.

Reply:
Going to leave this for the time being, with the intent to remove the `eventView` prop in an upcoming PR.